content/shared/influxdb3-admin/tables/create.md

```diff
@@ -0,0 +1,163 @@
+Use the [`influxdb3 create table` command](/influxdb3/version/reference/cli/influxdb3/create/table/)
+or the [HTTP API](/influxdb3/version/api/v3/) to create a table in a specified database in {{< product-name >}}.
+
+With {{< product-name >}}, tables and measurements are synonymous.
+Typically, tables are created automatically on write using the measurement name
+specified in line protocol written to InfluxDB.
```

Comment:
In all the monolith line protocol docs, we use "table" instead of measurement. Where we already call out the two as being synonymous, I think we can just use table here.

```suggestion
Typically, tables are created automatically on write using the table name
specified in line protocol written to InfluxDB.
```